README.md

```diff
@@ -155,6 +155,22 @@ require "super_diff/rspec"
 
 ## Configuration
 
+If you want to use something other than the default colors, you can
+configure them by adding them to your test helper file
+(`rails_helper` or `spec_helper`):
+
+``` ruby
+SuperDiff.configure do |config|
+  config.set_insertion_color(:green)
+  config.set_deletion_color(:red)
+  config.set_border_color(:yellow)
+  config.set_header_color(:yellow)
+end
+```
+
+See [lib/super_diff/csi/eight_bit_color.rb](eight_bit_color.rb) for the list
+of available colors.
+
 As capable as this library is,
```

Comment:
The Configuration section feels like it needs two subsections now. What do you think about adding a subheader here like `### Diffing custom objects`?